about.html

```diff
@@ -0,0 +1,153 @@
+        the 5-day event. <br>Various people active in the field of soccer will gather in one place to
+        share richer and more dynamical experiencies.<br> We hope that it will be an opportunity
+        to share our thoughts on creating an ecosystem.</p>
+          <div class="centerimg"><img src="img/about_icon.png" width=39 height=10 alt=""></div>
+        <p class="about3 tocenter2">September 14 - 18, 2022 (Wednesday - Sunday)
+          <br>See you at the FICA Soccer Tournament!</p>
+      </div>
+      <p class="about4 tocenter2">Do you have any other questions about FICA Soccer Tournament 2022?
+        <br class="onlymobile">&nbsp;&nbsp;&nbsp;
+        <a href="mailto:acolombo1@yahoo.com" class="asnormal">acolombo1@yahoo.com</a></p>
+      <div class="special onlymobile"><hr class="hr4"></div>
+      <div class="mini-spacer onlydesktop">&nbsp;</div>
+    </div>
+  </section>
+
+  <section class="about-section2-wrapper">
+    <div class="about-section2">
+      <h2 class="not-appear">FICA Global 2022 Logo Contest</h2>
+      <p class="about5 tocenter2">FICA Global 2022 Logo Contest</p>
+      <hr class="hr2">
+      <p class="about2 tocenter2">The winner of the logo contest, which was held from June 8th to July 7th,
+        was decided through online voting and evaluation by the judges.<br class="onlydesktop">
+        After consultation with the winning designer, the logo of FICA Tournament 2022, which was optimized
+        for use on the website and promotional materials, is as follows.</p>
+      <p class="tocenter2 margintopbottom"><img id="logo1" src="img/logo_big.png" width="260" height="102" Alt="Logo"></p>
+      <p class="about2 tocenter2">Aldo Colombo of Argentina, who designed the winning piece, explains the
+         meaning of the logo as follows:</p>
+      <p class="about2 tocenter2">"</p>
+      <p class="about2 tocenter2">The tri-colored ball symbolizes the integration of different
+        cultures and ethnicities, united in a sigle world of football. The
+        mimalist design symbolizes purity, harmony, wholeness and infinity.</p> 
+      <p class="about2 tocenter2">"</p>
+    </div>
+  </section>
+
+  <hr class="hr5">
+
+  <section class="about-section3-wrapper">
+    <div class="about-section3">
+      <h2 class="not-appear">View past FICA Tournaments</h2>
+      <p class="about5 tocenter2">View past FICA Tournaments</p>
+      <hr class="hr2">
+      <p class="about2 tocenter2">Take a look at the recent FICA Tournaments held in Ho Chi Min City,
+        Vietnam and Nassau, The Bahamas.</p>
+      
+      <div class="photos">
+        <img class="photo" id="2021" src="img/2021.jpg" width="451" height="252" Alt="2021 Tournament">
+        <img class="photo" id="2020" src="img/2020.jpg" width="451" height="252" Alt="2021 Tournament">
+      </div>
+      <div class="spacer">&nbsp;</div>
+    </div>
+  </section>
+  
+  <footer>
```

Comment:
- [ ] About Footer is almost ready, let's fix some details:

- The footer on the desktop version should have only one logo on the left and text on the right.

Template: (desktop)
![Captura de pantalla 2022-04-14 a la(s) 10 31 40 p  m](https://user-images.githubusercontent.com/83717285/163514449-a187be51-d169-4dc3-969f-b5285aa41e5e.png)

Your design: (desktop)
![Captura de pantalla 2022-04-14 a la(s) 10 31 00 p  m](https://user-images.githubusercontent.com/83717285/163514528-60d388f7-e829-4796-98fc-1b35c8a05375.png)

- The footer on the mobile version should have a light background with one logo on the left and text on the right.

Template: (mobile)
![Captura de pantalla 2022-04-14 a la(s) 10 31 17 p  m](https://user-images.githubusercontent.com/83717285/163514680-bc58e4ac-0345-4b0e-8606-7f5c5ab98046.png)

Your design:
![Captura de pantalla 2022-04-14 a la(s) 10 32 04 p  m](https://user-images.githubusercontent.com/83717285/163514703-8cd1e535-1a19-4e0e-b30d-59ee0adfb057.png)

Reply:
The same I was commenting on the index.html footer ...